filament/backend/src/opengl/GLUtils.h

```diff
@@ -29,6 +29,9 @@ namespace GLUtils {
 
 void checkGLError(utils::io::ostream& out, const char* function, size_t line) noexcept;
 void checkFramebufferStatus(utils::io::ostream& out, const char* function, size_t line) noexcept;
+#ifdef _MSC_VER
```

Comment:
Could you move this #define inside the #else below please? That or maybe we should just add it to `compiler.h` in the `utils` library.

Reply:
thanks for the suggestion @romainguy, this is done.